lualib/lua_selectors/transforms.lua

```diff
@@ -441,6 +441,28 @@ Empty string comes the first argument or 'true', non-empty string comes nil]],
     ['description'] = 'Extracts tld from a hostname represented as a string',
     ['args_schema'] = {}
   },
+  -- Converts list of strings to numbers and returns a packed string
+  ['pack_numbers'] = {
+    ['types'] = {
+      ['string_list'] = true
+    },
+    ['map_type'] = 'string',
+    ['process'] = function(inp, t, args)
+      local fmt = args[1] or 'f'
+      return fun.foldl(
```

Comment:
This is not an efficient approach - it it better to create a table and then apply table.concat (or use rspamd_text.fromtable function)